packages/presentation/lib/src/pages/login_page/bloc/login_data.dart

```diff
@@ -1,28 +1,40 @@
+import 'package:domain/domain.dart';
+
 class LoginData {
-  final String errorMessage;
   final String login;
   final String password;
+  final String? loginValidation;
+  final String? passwordValidation;
 
   const LoginData({
-    required this.errorMessage,
     required this.login,
     required this.password,
+    this.loginValidation,
+    this.passwordValidation,
   });
 
   LoginData copyWith({
-    String? errorMessage,
     String? login,
     String? password,
-  }) {
-    return LoginData(
-      errorMessage: errorMessage ?? this.errorMessage,
-      login: login ?? this.login,
-      password: password ?? this.password,
-    );
-  }
+    String? loginValidation,
+    String? passwordValidation,
```

Comment:
I don't think this fields should be in screen data. encapsulate them in exception object for ex. and keep them in bloc